ksp_plugin/interface.cpp

```diff
@@ -77,6 +84,118 @@ WXYZ ToWXYZ(Quaternion const& quaternion) {
           quaternion.imaginary_part().z};
 }
 
+// Similar to std::stod, but uses LOG(FATAL) instead of exceptions.
+double ParseDouble(std::string const& s, not_null<size_t*> size) {
+  char* interpreted_end;
+  char const* const c_string = s.c_str();
+  double result = std::strtod(c_string, &interpreted_end);
+  *size = interpreted_end - c_string;
+  CHECK_GT(*size, 0) << "invalid floating-point number " << s;
+  return result;
+}
+
+double ParseQuantity(std::string const& s, not_null<std::string*> unit) {
+  size_t i;
+  double magnitude = ParseDouble(s, &i);
+  for (*unit = ""; i < s.length(); ++i) {
+    if (!std::isspace(s[i])) {
+      *unit += s[i];
+    }
+  }
+  return magnitude;
+}
+
+// TODO(egg): actual parser.
```

Comment:
Can you create an issue for this and remove the comment?  It will be easier to track.